.github/workflows/eventlist.yml

```diff
@@ -161,6 +165,28 @@ jobs:
           test ! -s build/format-check.out
         working-directory: ./tools/eventlist
 
+  vulnerability-check:
+    name: "Vulnerability check"
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+        go: [1.19.3]

```

Comment:
Hm, we repeat the Go version multiple times. But I guess the very same version should be used in all places. Can we factor this version into a central place?

Reply:
[Addressed review comments](https://github.com/ARM-software/CMSIS-View/pull/14/commits/c0f043365bfd7f76b3f4cb4e2f9e6146237363b7)